napari/layers/labels/labels.py

```diff
@@ -311,14 +310,14 @@ def __init__(
             name = magic_name(data)
 
         self._seed = 0.5
-        self._seed_rng: Optional[int] = seed_rng
-        self._background_label = 0
-        self._num_colors = num_colors
         self._random_colormap = label_colormap(
-            self.num_colors, self.seed, self._background_label
+            49, self._seed, background_value=0
```

Comment:
maybe set this `49` to a variable and/or add a comment so we don't lose track of the origin and reason.

Reply:
Just gonna add a comment. I think this is minor and may change in the future (e.g. with a default colormap name) so I don't think it warrants its own variable. 😅